ELSpaceTests/TestDoubles/Stubs/ViewControllerTransitionContextStub.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+class ViewControllerTransitionContextStub: NSObject, UIViewControllerContextTransitioning {
+
+    var toView: UIView?
+    var fromView: UIView?
+
+    // MARK: - UIViewControllerContextTransitioning
+
+    var containerView = UIView()
+    var isAnimated = false
+    var isInteractive = false
+    var transitionWasCancelled = false
+    var presentationStyle: UIModalPresentationStyle = .none
+
+    func updateInteractiveTransition(_ percentComplete: CGFloat) {}
+
+    func finishInteractiveTransition() {}
+
+    func cancelInteractiveTransition() {}
+
+    func pauseInteractiveTransition() {}
+
+    func completeTransition(_ didComplete: Bool) {}
```

Comment:
Please implement this function

```swift
private(set) var invokedCompleteTransition: (count: Int, didComplete: Bool)?

...

func completeTransition(_ didComplete: Bool) {
    var count = invokedCompleteTransition?.count ?? 0
    count += 1

    invokedCompleteTransition = (count: count, didComplete: didComplete)
}

```

And please write additional tests whether completeTransition was called with correct parameter.